src/OpenSage.Game/Content/ModelLoader.cs

```diff
@@ -319,17 +319,32 @@ private static FixedFunction.VertexMaterial[] CreateMaterials(W3dMesh w3dMesh)
             {
                 vertices[i] = new MeshVertex.Basic
                 {
-                    Position = w3dMesh.Vertices.Items[i],
-                    Normal = w3dMesh.Normals.Items[i],
+                    Position0 = w3dMesh.Vertices.Items[i],
+                    Position1 = w3dMesh.Vertices2 != null
```

Comment:
Is this necessary, or can you fallback to Vector3.Zero? Or put another way, if there are two bone influences, is Positions2 always present?

Reply:
Yes the second position is always there as it's the same vertex but in the other bone's space.

Reply:
well how would you test that for every model, need to iterate over every model and over every vertex influence?
Also what is the problem to use the default vertices here?

Reply:
The way they apply the bone influences wouldn't even work without a corresponding set of vertices.

Reply:
`		int index = blendIndices.x * 2;
		WorldPosition = BoneTransformPosition(WorldBones[index], WorldBones[index+1], InSkin.Position0) * InSkin.BlendWeights.x;
		index = blendIndices.y * 2;
		WorldPosition += BoneTransformPosition(WorldBones[index], WorldBones[index+1], InSkin.Position1) * InSkin.BlendWeights.y;`

Reply:
yea looks like it, wrote some code to test it though, so if the second bone index is not 0, there are always vertices2 and normals2